src/Hl7.Fhir.Specification.Tests/TestData/summary-test/TestXmlRepresentation.StructureDefinition.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <url value="http://hl7.org/fhir/StructureDefinition/TestXmlRepresentation" />
+  <name value="TestXmlRepresentation" />
+  <status value="draft" />
+  <fhirVersion value="4.0.0" />
+  <kind value="logical" />
+  <abstract value="false" />
+  <type value="TestXmlRepresentation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/DomainResource" />
+  <derivation value="specialization" />
+  <snapshot>
+    <element id="TestXmlRepresentation">
+      <path value="TestXmlRepresentation" />
+      <min value="0" />
+      <max value="*" />
+    </element>
+    <element id="TestXmlRepresentation.XmlAttr">
```

Comment:
Paths with capital letters? Meaning element names starting with a capital letter?  This will be a problem, since the FhirXmlParser will assume this is a nested resource (in FHIR, only contained resources will have capital inital letters, so there is code depending on this)

Reply:
Turns out this is just a test.